src/api/curriculumApi.ts

```diff
@@ -71,22 +140,88 @@ export async function fetchCompetenceGoal(
   const json: CompetenceGoal = await resolveJson(response);
   return {
     id: json.id,
+    title: filterTextsForLanguage(json.tittel.tekst, context.language),
+    type: 'LK20',
     code: json.kode,
-    title: filterTitleForLanguage(json.tittel.tekst, context.language),
     curriculum: mapReference(json['tilhoerer-laereplan']),
     competenceGoalSet: mapReference(json['tilhoerer-kompetansemaalsett']),
     crossSubjectTopics: mapElements(json['tilknyttede-tverrfaglige-temaer']),
     coreElements: mapElements(json['tilknyttede-tverrfaglige-temaer']),
   };
 }
 
-export async function fetchCompetenceGoals(
+export async function fetchLK20CompetenceGoals(
+  codes: string[],
+  context: Context,
+): Promise<GQLCompetenceGoal[]> {
+  return Promise.all(codes.map(code => fetchLK20CompetenceGoal(code, context)));
+}
+
+export async function fetchCoreElement(
+  code: string,
+  context: Context,
+): Promise<GQLCoreElement> {
+  const response = await fetch(
+    `https://data.udir.no/kl06/v201906/kjerneelementer-lk20/${code}`,
+    context,
+  );
+  const json: CoreElement = await resolveJson(response);
+  return {
+    id: json.id,
```

Comment:
Bruk koden her også.
```suggestion
    id: json.kode,
```